styles/layout.css

```diff
@@ -55,7 +54,7 @@ body{
 
 .nav_menu a{
     color: white;
-    font-size: 1.5rem;
+    font-size: 1rem;
 }
 
 .blur-container {
```

Comment:
Fix the typo in the HTML and this should work perfectly

Reply:
change done